test/aws-kinesis/aws-kinesis-client.yaml

```diff
@@ -10,10 +10,14 @@ spec:
   containers:
     - name: aws-kinesis-client
       image: 'amazon/aws-cli:2.1.10'
-      command: ["aws", "kinesis", ${aws.kinesis.command}]
+      command: ["/bin/sh", "-c"]
```

Comment:
I think we should use env settings on the container spec (see https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)

With that we can keep using the `aws` command instead of `/bin/sh` with `export` arguments.

Reply:
Great idea! Thanks for the hint, I'll change it accordingly.